VideoApi/VideoApi.DAL/Migrations/20220405223938_AddTrackingDateTimeColumns.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace VideoApi.DAL.Migrations
+{
+    public partial class AddTrackingDateTimeColumns : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTime>(
+                name: "Timespan",
+                table: "TestCallResult",
+                nullable: true);
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: "CreatedAt",
+                table: "RoomParticipant",
+                nullable: true);
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: "UpdatedAt",
+                table: "RoomParticipant",
+                nullable: true);
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: "CreatedAt",
+                table: "RoomEndpoint",
+                nullable: true);
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: "UpdatedAt",
+                table: "RoomEndpoint",
+                nullable: true);
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: "CreatedAt",
+                table: "Room",
+                nullable: true);
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: "UpdatedAt",
+                table: "Room",
+                nullable: true);
+
+            migrationBuilder.AlterColumn<Guid>(
```

Comment:
Why is this needed? Do we know for sure that this value is not currently null in all databases?